### PR TITLE
Added `create-window` sample 

### PR DESCRIPTION
**Pull request by Ericvf**
This sample project was found in the old repository for the _Rust for Windows_ samples.

### CODE REVIEW

**Review by kennykerr — CHANGES_REQUESTED** [review]
Thanks, happy to restore my old sample if its helpful. 

Please just run `cargo fmt` and then the build should be satisfied.

**Comment by Ericvf** [comment]
Thanks. I've commit the changes made from `cargo fmt`

**Review by riverar — DISMISSED** [review]

File: create_window/bindings/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "bindings"
+version = "0.0.0"
+edition = "2018"
+
+[dependencies]
+windows = { version = "0.18"}
```
Comment (kennykerr) [inline create_window/bindings/Cargo.toml:7]:
```suggestion
windows = "0.18"
```

**Review by kennykerr — DISMISSED** [review]

File: create_window/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "create-window"
+version = "0.0.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bindings = { path = "bindings" }
+windows = { version = "0.18" }
```
Comment (kennykerr) [inline create_window/Cargo.toml:10]:
```suggestion
windows = "0.18"
```

File: create_window/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "create-window"
+version = "0.0.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
```
Comment (kennykerr) [inline create_window/Cargo.toml:7]:
```suggestion
```

**Pull request merged**